src/assets/scss/dropdown.scss

```diff
@@ -83,8 +83,9 @@
     }
 }
 
-.selected {
-    pointer-events: none;
+.selected,
+.disabled {
```

Comment:
What happens if this class does not exist?

Reply:
The arrow down icon still interact with mouse-hover, also i don't want  to "selected" class be apply anyone besides the "dropdown-item" and "pointer-events: none !important;" deal with the mouse-hover persistence.

Reply:
I have remove this class, i found a better way to do this.